app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt

```diff
@@ -75,16 +95,26 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
                 }
             }
         },
-        textStyle = LocalTextStyle.current.copy(fontSize = 14.sp, textAlign = TextAlign.Start),
-        placeholderTextStyle = LocalTextStyle.current.copy(fontSize = 14.sp, textAlign = TextAlign.Center),
+        interactionSource = interactionSource,
+        textStyle = textStyle.copy(fontSize = 14.sp),
```

Comment:
I don't remember if we have already some `WireDimensions` for these values, if we have one would be a good idea to use it